include/roboteam_ai/world_new/Robot.hpp

```diff
@@ -140,8 +140,8 @@ class Robot {
     [[nodiscard]] unsigned long getLastUpdatedWorldNumber() const noexcept;
 
    public:
-    explicit Robot(std::unordered_map<uint8_t, proto::RobotFeedback> &feedback, const proto::WorldRobot &copy, Team team = both, unsigned char dribblerState = 0,
-                   unsigned long worldNumber = 0);
+    explicit Robot(std::unordered_map<uint8_t, proto::RobotFeedback> &feedback, const proto::WorldRobot &copy, Team team = both,
+                   std::optional<rtt::world_new::view::BallView> ball = std::nullopt, unsigned char dribblerState = 0, unsigned long worldNumber = 0);
```

Comment:
Why is the dribblerState an unsigned char?

Reply:
I don't know, it wasn't my decision. It is already this way in the "old" world.

Reply:
Strange